eng/templates/jobs/build-from-source.yml

```
@@ -0,0 +1,130 @@
+parameters:
+  # Specifies the prefix for source-build jobs added to pipeline. Use this if disambiguation is necessary.
+  jobNamePrefix: 'sourcebuild'
+
+  # Specifies that the repo only builds managed components. This indicates that output is not RID-specific 
+  # and only one build job is required. A linux x64 machine will be used, but the build must continue to 
+  # work on all supported platforms equally.
+  # If false, pecifies that the repo has native components that can link to platform-provided libraries.
+  #   - Adds additional portable job(s) for supported portable platforms
```

I think we should assume the lack of `win` and non-`x64` RIDs is temporary and go with the intent here for readability:

```suggestion
  #   - Adds additional portable jobs for supported portable platforms
```